### DISCUSSION TITLE
LX-R 120 Issues

### DISCUSSION THREAD

**Discussion by FanRookie** [EvanMulawski/FanControl.CorsairLink#216] [Q&A] [answered]
I saw the LX-120 is compatible, are the LX-R? Keep getting this error message after installing the new fans. 
[log.txt](https://github.com/user-attachments/files/18268268/log.txt)

**Comment by EvanMulawski** [comment]
Hi @FanRookie, can you post the CorsairLink.log file? Thanks!

**Reply by FanRookie** [reply]
[CorsairLink.log](https://github.com/user-attachments/files/18271234/CorsairLink.log)

**Reply by EvanMulawski** [reply, marked as answer]
Try [v1.7.0-beta.8](https://github.com/EvanMulawski/FanControl.CorsairLink/releases/tag/v1.7.0-beta.8)

**Reply by FanRookie** [reply]
Updated to Net8 and installed the beta and it seems to be working now. Appreicate the quick response and the help!
